dev-libs/libdaemon/libdaemon-0.14-r4.ebuild

```diff
@@ -27,40 +27,33 @@ src_prepare() {
 	# (elibtoolize is insufficient)
 	# bug #668404
 	eautoreconf
-
-	# doxygen is broken with out-of-source builds
-	multilib_copy_sources
 }
 
-multilib_src_configure() {
+src_configure() {
 	econf \
 		--localstatedir=/var \
 		--disable-examples \
-		--disable-lynx \
-		$(use_enable static-libs static)
+		--disable-lynx
 }
 
-multilib_src_compile() {
+src_compile() {
 	emake
 
-	if multilib_is_native_abi && use doc; then
+	if use doc; then
+		HTML_DOCS=( doc/README.html doc/style.css doc/reference/html/. )
+
 		einfo "Building documentation"
 		emake doxygen
 	fi
 }
 
-multilib_src_install() {
-	emake DESTDIR="${D}" install
+src_install() {
+	default
 
-	if multilib_is_native_abi && use doc; then
-		docinto html
-		dodoc -r doc/README.html doc/style.css doc/reference/html/*
+	if use doc; then
+		einstalldocs
 		doman doc/reference/man/man3/*.h.3
 	fi
```

Comment:
change this block to
`use doc && doman doc/reference/man/man3/*.h.3`
the einstalldocs isn't necessary